docs/index.rst

```diff
@@ -0,0 +1,14 @@
+
+FoxPuppet
+=========
+
+FoxPuppet is a tool used for testing the `Firefox <https://www.mozilla.org/en-US/firefox/products/>`_
```

Comment:
We're using it for testing, but I think it's better to say it's for automating Firefox. In the README we say:

> Fox Puppet is a library for desktop firefox that provides ui interaction and state using the Selenium library.

I think we need to be consistent, and would suggest the following for both:

> FoxPuppet is a library for automating user interactions in [Firefox](https://www.mozilla.org/en-US/firefox/desktop/) using [Selenium](http://seleniumhq.org/).